Jenkinsfile.public

```diff
@@ -105,6 +108,17 @@ pipeline {
                         sh 'make test-sim-backwards-compatible'
                     }
                 }
+
+                stage('npm') {
+                    steps {
+                        dir("web/satellite") {
+                            sh 'npm run build'
```

Comment:
lets also add `npm run test` to run unit tests

Reply:
Let's try, I hope it won't be too slow.